app/controllers/state_file/questions/questions_controller.rb

```diff
@@ -35,6 +35,14 @@ def state_code
         state_code_
       end
 
+      def state_name
```

Comment:
Are there other possible applications of this method now that it's here?

Reply:
Factoring this out into a method doesn't seem worth it to me if it is only used in one place

Reply:
Yeah, the params are used everywhere in the views.  I wanted to update all those places using this.

Reply:
I will add another PR making those updates.

Reply:
https://github.com/codeforamerica/vita-min/pull/4272